### PR TITLE
Doc move to github for Instagram Authenticator

### PR DESCRIPTION
**Pull request by nalakas**
Doc move to github for Instagram Authenticator

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+* [Configuring the service provider](#Configuring-the-service-provider)
+* [Configuring claims](#Configuring-claims)
+* [Configuring requested claims for travelocity.com](#Configuring-requested-claims-for-travelocity.com)
+* [Testing the sample](#Testing-the-sample)
+
+## Deploying Instagram artifacts
+
+* Either Download the artifacts for this authenticator from [the store](https://store.wso2.com/store/assets/isconnector/details/175db9b2-1aae-4402-adee-94c4acd751d2) or [build the project](Building From the Source) to get the jar and by adding it in the <IS-Home>/repository/components/dropins directory.
+
+* If you want to upgrade the Github Authenticator (.jar) in your existing IS pack, follow the bellow instructions.
```
Comment (menakaj) [inline docs/README.md]: Github authenticator? or Instagram?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  * Stop WSO2 Identity Server if the server is already running.
+  * Download and extract the latest version of the connector artifacts (.jar, .war, gadgets etc.,) from the connector store.
```
Comment (menakaj) [inline docs/README.md]: connector artifacts === authenticator artifacts.

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+  * Replace the old .jar file found in the <IS_HOME>/repository/components/dropins folder with the new .jar file that you downloaded.
```
Comment (menakaj) [inline docs/README.md]: Mention the name of the .jar file. we can use wildcards for version. i.e org.wso2.carbon.extension.identity.authenticator.outbound.instagram-x.x.jar

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+* Log in to the [management console](https://docs.wso2.com/display/IS570/Getting+Started+with+the+Management+Console) as an administrator.
+
+* In the `Identity Providers` section under the `Main` tab of the management console, click `Add`.
+
+* Give a suitable name for `Identity Provider` Name and configure the authenticator. To do this, navigate to `Instagram Configuration` under `Federated Authenticators` and fill the form.
+    ![3](images/instagramIDP.png "instagramIDP.png")
+ 
+    Do the following configurations.
+    
+    |Field|Description|SAmple value|
```
Comment (menakaj) [inline docs/README.md]: Typo... should be: Sample Value

**Review by menakaj — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (nalakas) [reply docs/README.md]: fixed

File: docs/README.md (hunk 2 above)
Comment (nalakas) [reply docs/README.md]: fixed

File: docs/README.md (hunk 3 above)
Comment (nalakas) [reply docs/README.md]: fixed

File: docs/README.md (hunk 4 above)
Comment (nalakas) [reply docs/README.md]: fixed

**Review by menakaj — APPROVED** [review]

**Pull request merged**